semconv-incubating/src/main/java/io/opentelemetry/semconv/incubating/HttpIncubatingAttributes.java

```diff
@@ -112,7 +127,12 @@ public final class HttpIncubatingAttributes {
   public static final AttributeKey<Long> HTTP_REQUEST_RESEND_COUNT =
       longKey("http.request.resend_count");
 
-  /** Deprecated, use {@code http.request.header.content-length} instead. */
+  /**
+   * Deprecated, use {@code http.request.header.content-length} instead.
+   *
+   * @deprecated Deprecated, use `http.request.header.content-length` instead.
```

Comment:
This isn't a showstopper, but it does really a little redundant.

Reply:
Good point, but we don't control that text. It comes directly from the semantic conventions YAML: https://github.com/open-telemetry/semantic-conventions/blob/cafda7127683b7f667e27cdbd3220510b6f998c9/model/registry/deprecated/http.yaml#L34